### PR TITLE
Add legacy data_access_point

### PR DESCRIPTION
**Pull request by ilkilic**
- add data_access_point for legacy purposes
- update example readme
- fix create_venv.sh

### CODE REVIEW

File: examples/emodel_pipeline_local_python/scripts/create_venv.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ cd software
 ../$VENV/bin/pip install wheel
 
 git clone https://github.com/BlueBrain/BluePyEModel.git
-../$VENV/bin/pip install -e ./bluepyemodel
```
Comment (AurelienJaquier) [inline examples/emodel_pipeline_local_python/scripts/create_venv.sh:17]: You don't need to change that, I think

Comment (ilkilic) [reply examples/emodel_pipeline_local_python/scripts/create_venv.sh:17]: when you clone it, the folder name has the uppercase letters

Comment (AurelienJaquier) [reply examples/emodel_pipeline_local_python/scripts/create_venv.sh:17]: oh yeah, you are right

**Review by darshanmandge — APPROVED** [review]
Thanks for updating the example and adding the argument data_access_point. It looks good now.

**Review by AurelienJaquier — APPROVED** [review]
looks great, thanks!

**Pull request merged**